lighthouse-core/test/audits/preload-lcp-image-test.js

```diff
@@ -132,4 +121,15 @@ describe('Performance: preload-lcp audit', () => {
     expect(results.notApplicable).toBeTruthy();
     expect(results.details).toBeUndefined();
   });
+
+  it('should suggest preloading a lcp image if all criteria is met', async () => {
```

Comment:
this test seems to be identical to the removed one unless I'm reading something wrong?

we should get tests in here that exercises the new branch. basically two cases to handle the different branches of the `Math.max`

- a dependency download took less time than LCP node took (increase LCP node bytes to simulate this)
- a dependency download took more time than LCP node took (increase dependency node bytes to simulate this)

Reply:
Yeah, that was just me moving the test around. Gonna add the tests you mentioned.